main.py

```diff
@@ -1,80 +1,58 @@
 import base64
-import httplib2
-import oauth2
 import random
 from xml.etree import ElementTree as etree
-from urllib.parse import urlencode
 import json
 import os
+import requests
+from requests_oauthlib import OAuth1
+import time
 
 CONSUMER_KEY = os.getenv("CONSUMER_KEY")
 CONSUMER_SECRET = os.getenv("CONSUMER_SECRET")
 ACCESS_KEY = os.getenv("ACCESS_KEY")
 ACCESS_SECRET = os.getenv("ACCESS_SECRET")
+FLICKR_API_KEY = os.getenv("FLICKR_API_KEY")
 
 
-def flickr_query_to_get_list_of_pictures(request_h):
-    """
-    function to get list of photos
-    :param request_h: http stream
-    :return: list of bytestrings for each photo
-    """
+def flickr_get_photos_list():
     payload = {'method': 'flickr.people.getPublicPhotos',
-               'api_key': '1a19ecf3ddf46b3323bca01dc8afbd81',
+               'api_key': FLICKR_API_KEY,
                'user_id': '76163487@N05'
                }
-    resp, content = request_h.request('https://api.flickr.com/services/rest/?&{}'.format(urlencode(payload)), "POST")
-    xml = etree.fromstring(content.decode('utf-8'))
-    xml = xml[0]  # get child of root (list of <photo> tags)
-    photos = [{'id': tag.attrib['id'], 'secret': tag.attrib['secret'],
-               "farm": tag.attrib["farm"], "server": tag.attrib["server"]} for tag in xml]
-    return photos
+    photos_list_response = requests.post('https://api.flickr.com/services/rest', data=payload)
+    return [tag.attrib for tag in etree.fromstring(photos_list_response.text)[0]]
 
 
-def flickr_query2(request_h2, photo_index):
-    """
-    function to load photo from server
-    :rtype: object
-    """
-    resp2, content2 = request_h2.request('https://farm{}.staticflickr.com/{}/{}_{}.jpg'.format(
-        photo_index["farm"], photo_index["server"], photo_index['id'], photo_index["secret"]))
-    return content2
+def flickr_get_photo(photo_attributes):
+    flickr_binary_file = requests.get('https://farm{}.staticflickr.com/{}/{}_{}.jpg'.format(
```

Comment:
You can use [f-strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings). Although in this case it will probably be less readable :)

Reply:
I will read about f-strings, and I agree in that case will be better leave string as it is now